src/providers/run-test-code-lens-provider.ts

```diff
@@ -18,7 +21,7 @@ function walkMatches(text: string, callback: Function) {
 
 export class RunTestCodeLensProvider implements CodeLensProvider {
   async provideCodeLenses(document: TextDocument): Promise<CodeLens[]> {
-    if (document.fileName.match(/-test.(js|ts)$/) === null) {
+    if (document.fileName.match(/-test.(js|ts|gjs|gts)$/) === null) {
```

Comment:
Without `typescript` language support, `*.ts` test files will not render the lens.

Reply:
Oh nope. I added language support for `typescript` a few lines above if that's what you meant.

Appreciate the reply from the living legend.